src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery extends ColorSupplier {
```

Comment:
```suggestion
public class Lottery {
```

Reply:
oh, I see. But when I'm trying to run it without 'extends ColorSupplier' I have compiler error. Here are the screenshot
![Extends_Issue](https://github.com/mate-academy/jv-lottery/assets/110618327/00fdfb01-fbd0-446d-9ab0-99ba8a7edb6f)